enterprise/admin/__init__.py

```diff
@@ -15,9 +15,9 @@
 from enterprise.admin.utils import UrlNames
 from enterprise.admin.views import EnterpriseCustomerManageLearnersView
 from enterprise.django_compatibility import reverse
-from enterprise.models import (
+from enterprise.models import (  # pylint:disable=no-name-in-module
     EnterpriseCustomer, EnterpriseCustomerUser, EnterpriseCustomerBrandingConfiguration,
-    EnterpriseCustomerIdentityProvider,
+    EnterpriseCustomerIdentityProvider, HistoricalUserDataSharingConsentAudit
```

Comment:
Only one `UserDataSharingConsentAudit` is attached to a given `EnterpriseCustomerUser` at a given time. We want to show a list of the changes that have been made to a given `UserDataSharingConsentAudit`, rather than just the current state, which means we need to use the historical model.